kymatio/scattering3d/frontend/numpy_frontend.py

```diff
@@ -20,8 +20,7 @@ def build(self):
         ScatteringBase3D.create_filters(self)
 
     def scattering(self, input_array):
-        if not type(input_array) is np.ndarray:
-            raise TypeError('The input should be a NumPy array.')
+        self.backend.input_checks(input_array)
```

Comment:
this is to let jax arrays through as well?
I like the change, just need to understand the motivation. I would have assumed that since in the jax case, `np = jax.numpy`, it should all be transparent